types/src/v17/mining/into.rs

```diff
@@ -17,6 +17,7 @@ impl GetBlockTemplate {
         use GetBlockTemplateError as E;
 
         let version = block::Version::from_consensus(self.version);
+        let capabilities = self.capabilities.into_iter().map(|s| s.to_string()).collect::<Vec<_>>();
```

Comment:
This is mapping vector of `String` to a vector of `String`, we can just use `self.capabilities` directly below.